Automattic/wp-calypso
# WordAds Stats: fix broken links in Odyssey

#### Proposed Changes

Restores clicking behaviours on the Ads page:

* Restore information icon clicking
* Bars clicking
* Tabs clicking

We should avoid use `window.location` object directly for Stats, as in Odyssey the paths are actually hashes. We should use `context.path`, `context.query` and `context.pathname` instead, which are injected by `page.js`.

#### Testing Instructions

* Verify Odyssey with Option 1 in `PejTkB-3E-p2`
* Enable WordAds - `/wp-admin/admin.php?page=jetpack#/traffic?term=wordads`
* Open `/wp-admin/admin.php?page=stats&flags=stats%2Fads-page`
* Ensure Ads page generally works
* Ensure the click actions for bars, tabs and information icon works
* Open live branch link
* Enusre Ads page work for WPCOM

![image](https://user-images.githubusercontent.com/1425433/214210380-b2c024b6-2b09-411e-929a-72f2397cce21.png)

#### Pre-merge Checklist
